packages/back-end/src/controllers/config.ts

```diff
@@ -84,7 +84,7 @@ type CompressedExperimentOptions = {
 const baseScript = fs
   .readFileSync(path.join(__dirname, "..", "templates", "javascript.js"))
   .toString("utf-8")
-  .replace(/.*eslint-.*\n/g, "")
+  .replace(/\/\*\s*eslint-.*\*\//, "")
```

Comment:
Not sure if this is relevant to our use cases since you mentioned there's a template (do we control this template?), but line-style comments are valid for eslint too (e.g. `// eslint-disable-next-line`). My thoughts are the original was trying to capture that as well. Could we possibly miss some?

Reply:
That's a good call ... From what I can tell there aren't any line-style comments. The template is located at `packages/back-end/src/templates/javascript.js`. I did a search for `//` and couldn't find anything, and since it's uglified and minified, I'm doubtful that there are any.